process/transactions/scDeploysProcessor.go

```diff
@@ -6,12 +6,13 @@ import (
 	"encoding/hex"
 
 	"github.com/ElrondNetwork/elastic-indexer-go/data"
-	"github.com/ElrondNetwork/elrond-go/core"
-	"github.com/ElrondNetwork/elrond-go/hashing/keccak"
-	"github.com/ElrondNetwork/elrond-go/process/factory"
-	"github.com/ElrondNetwork/elrond-vm-common/parsers"
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/hashing/keccak"
 )
 
+// ArwenVirtualMachine is a byte array identifier for the smart contract address created for Arwen VM
+var ArwenVirtualMachine = []byte{5, 0}
```

Comment:
move to core, please

Reply:
This const will be removed in the next PR where sc deploys processor was refactored.

Reply:
great